### PR TITLE
Implement basic web server

### PR DESCRIPTION
**Pull request by JamesLee-Jones**
Implements a basic Flask web server.

Fixes: #1

### CODE REVIEW

**Review by AbdulRJacob — APPROVED** [review]

File: app.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():  # put application's code here
+    return 'Hello World!'
+
+
```
Comment (zeegtab) [inline app.py:10]: nit: Unnecessary line

File: app.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
```
Comment (zeegtab) [inline app.py:5]: nit: Unnecessary line

**Review by dave5202 — APPROVED** [review]

**Review by zeegtab — CHANGES_REQUESTED** [review]
Please fix.

File: app.py (hunk 1 above)
Comment (JamesLee-Jones) [reply app.py:10]: In python, double line spaces are preferable.

File: app.py (hunk 2 above)
Comment (JamesLee-Jones) [reply app.py:5]: Same as above.

**Review by ESummerscale — APPROVED** [review]

**Pull request merged**